CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ UNRELEASED
 
 * [ [#1227](https://github.com/digitalfabrik/integreat-cms/issues/1227) ] Correct URL and Path field in imprint API
 * [ [#1222](https://github.com/digitalfabrik/integreat-cms/issues/1222) ] Fix missing translations and archived pages in API
+* [ [#1226](https://github.com/digitalfabrik/integreat-cms/issues/1226) ] Skip cache for childen and parents end points
```

Comment:
```suggestion
```

Reply:
Where the hack is this coming from?! I need to investigate what is going on here. This happens whenever I rebase on develop.

Reply:
Maybe you have [rerere](https://git-scm.com/book/de/v2/Git-Tools-Rerere) enabled and git has remembered how you resolved this conflict the first time and repeats it now everytime it sees it again :see_no_evil: